src/components/buttons/Button.module.css

```diff
@@ -74,4 +81,26 @@
     .gray {
         padding: 1.4vh 0.8vw;
     }
+}
+
+@media (max-width: 1000px) {
+    .green,
+    .gray {
+        margin: 1vh 0;
+        min-width: 23vw;
+        text-align: center;
+        padding: 1.75vh 2vw;
+        display: flex;
+        justify-content: center;
+    }
+}
+
+@media (max-width: 576px) {
+    .gray {
+        padding: 2vh 2vw;
+    }
+    .green {
+        padding: 1.75vh 2vw;
+        order: -1;
+    }
 }
```

Comment:
GitHub needs empty line at the end of each file you want to add.
Add it, as I mentioned before